controllers/neutronapi_controller.go

```diff
@@ -854,6 +858,16 @@ func (r *NeutronAPIReconciler) generateServiceConfigMaps(
 	templateParameters["NBConnection"] = dbmap["internal-NB"]
 	templateParameters["SBConnection"] = dbmap["internal-SB"]
 
+	// TODO(ihar): make the metadata config secret too
+	templateParameters["TransportURL"] = string(transportURLSecret.Data["transport_url"])
+
+	// TODO(ihar): when is it not localhost?
+	templateParameters["MetadataBindHost"] = "127.0.0.1"
+
+	// TODO(ihar): set to correct values from nova - to figure out how to extract them from nova?
+	templateParameters["NovaMetadataHost"] = "1.2.3.4"
+	templateParameters["MetadataProxySharedSecret"] = "1234567842"
```

Comment:
While this data coming from the osp-secret both nova- and neutron-operator gets, the password selector is configuration of the Nova CR so this data also depends on that password selector value. As this can be per cell and neutron has no cell concept this secret value should be provided by the same mechanism as the metadata host.